server/aesdsocket.c

```diff
@@ -491,16 +528,15 @@ int main(int argc, char *argv[])
         goto exit_segment;
 	}
 
-    // Creating output file
-    // NOTE: fd is a global variable and is not supplied as param to each thread
-    fd = open(FILE_NAME, O_RDWR | O_CREAT, 0644);
+#ifndef USE_AESD_CHAR_DEVICE
```

Comment:
@SundarKrishnakumar why would this code remain in the `#ifndef USE_AESD_CHAR_DEVICE` case?  A comment about this would be good.